### ISSUE TITLE
Auto-Magic Handle Lint/Format Issues

### ISSUE THREAD

**Issue by edm00se** [edm00se/awesome-board-games#100]
TL;DR: Looking to implement [samuelmeuli/lint-action](https://github.com/samuelmeuli/lint-action) or similar.

## Situation

A contributor, not me, submits a high quality PR with their awesome board game. They used the web editor and missed a single character (some small amount of variance) causing the linter to complain and block the PR. This is 💩, since it's 99% awesome.

## Why This Is Needed

My previous attempt (#62) worked well on `master` in this repo, but wasn't able to handle PRs as gracefully as desired. In short, it would require formatter/linting checking _after merging into `master`, causing additional commits. This is inefficient with the pattern of git and GH PRs, as we're meant to detect and address this stuff _before_ merging.

**Comment by edm00se** [comment]
For the particular gh action mentioned above, it may need to wait for this to be resolved: samuelmeuli/lint-action#13

**Comment by edm00se** [comment]
TL;DR: this doesn't seem easily done, especially not for while the code is within a forked (non-source) repo
